pom.xml

```diff
@@ -163,7 +178,7 @@
         <dependency>
             <groupId>com.github.Emibergo02</groupId>
             <artifactId>ConfigLib</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>4.2.0</version>
```

Comment:
I think we can move the project to Java 17 and use [https://github.com/Exlll/ConfigLib]( https://github.com/Exlll/ConfigLib) directly.
As you can see [there](https://bstats.org/plugin/bukkit/RedisEconomy/16802) no one is using a mc version below 1.17.1

Reply:
no please use the dependency below as indicated in the link
```xml
<dependency>
    <groupId>com.github.Exlll.ConfigLib</groupId>
    <artifactId>configlib-yaml</artifactId>
    <version>v4.3.0</version>
</dependency>
```